src/locales/nl-NL/stream.ftl

```diff
@@ -0,0 +1,239 @@
+### Localization for Embed Stream
+
+## General
+
+general-userBoxUnauthenticated-joinTheConversation = Log in of Registreer je in 1 minuut om een reactie te kunnen plaatsten
+general-userBoxUnauthenticated-signIn = Inloggen
+general-userBoxUnauthenticated-register = Registreren
+
+general-userBoxAuthenticated-signedInAs =
+  Ingelogd als <Username></Username>.
+
+general-userBoxAuthenticated-notYou =
+  Bent u dit niet? <button>Uitloggen</button>
+
+general-tabBar-commentsTab = Reacties
+general-tabBar-myProfileTab = Mijn profiel
+general-tabBar-configure = Instellingen
+
+## Comments Tab
+
+comments-allCommentsTab = Alle reacties
+comments-featuredTab = Uitgelicht
+comments-featuredCommentTooltip-how = Hoe wordt een reactie uitgelicht?
+comments-featuredCommentTooltip-handSelectedComments =
+  Reacties zijn handmatig door ons gekozen als het lezen waard.
+comments-featuredCommentTooltip-toggleButton =
+  .aria-label = Uitgelichte reacties informatie knop
+
+
+comments-streamQuery-storyNotFound = Bericht niet gevonden
+
+comments-postCommentForm-submit = Plaats reactie
+comments-replyList-showAll = Toon alles
+comments-replyList-showMoreReplies = Laad meer reacties
+
+
+comments-viewNew =
+  { $count ->
+    [1] Nieuwe reactie {$count}
+    *[other] Nieuwe reactie {$count}
+  }
+comments-loadMore = Meer laden
+
+comments-permalinkPopover =
+  .description = A dialog showing a permalink to the comment
```

Comment:
Did you forget this line? English is already the fallback language.

Reply:
fixed